pkgs/by-name/ar/ardopc/package.nix

```diff
@@ -0,0 +1,47 @@
+{ fetchFromGitHub
+, lib
+, makeWrapper
+, pkg-config
+, stdenv
+, alsa-lib
+, flrig
+, hamlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ardopc";
+  version = "unstable-2021-08-28";
+
+  src = fetchFromGitHub {
+    owner = "hamarituc";
+    repo = "ardop";
+    rev = "20210828";
+    hash = "sha256-OUw9spFTsQLnsXksbfl3wD2NyY40JTyvlvONEIeZyWo=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/ARDOPC";
+
+  installPhase = ''
+    install -D ardopc $out/bin/ardopc
+  '';
```

Comment:
Can you move this after buildInputs and nativeBuildInputs?

Also it would be nice to  `runHook preInstall` and `runHook postInstall`